Workload-Security/Support/ReadMe.md

```diff
@@ -0,0 +1,49 @@
+# Trend Cloud One Workload Security - Agent Diagnostic Collection Tool
+
+This tool is intended to aid in generating and collecting the workload security agent diagnostic log package.
+The output of this tool can be provided to Trend Micro Support on case creation.
+
+---
+
+## Requirements for use.
+
+### SSM Automation Operations IAM Role
+- Create an IAM Role for SSM Automations to Assume to execute document. See example [SSM-Policy](https://github.com/JustinDPerkins/TrendCloudOne-SupportCollection/blob/main/Workload-Security/aws/ssm-iam-example-policy.json)
+- The IAM Role requires SSM to be a Trusted Entity.
+
+### Instance Requirements:
+- Requires the Workload Security Agent to be deployed on instance.
+- Requires the SSM Agent to be Installed and Running. See [here](https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html)
+- The EC2 Instance requires an IAM Role with SSMManagedCore permissions.
```

Comment:
`SSMManagedCore` permissions to what resource?

Reply:
This an AWS Managed Policy provided by AWS. It contains the bare minimum to allow SSM to function. This needs to be apart of an EC2 IAM Policy to work with Systems Manager in general. This is expected to already be in place by customers leveraging SSM already for other tasks.

I attached AWS Doc link to further clarify.